frontend/src/components/Pages/InspectionPage/InspectionOverview.tsx

```diff
@@ -123,7 +112,15 @@ export const InspectionOverviewSection = () => {
                                 <AddPredefinedMissionsButton />
                                 <CreateEchoMissionButton />
                             </StyledButtons>
-                            {allMissions && <AllInspectionsTable inspections={allMissions} />}
+                            {allInspections && allInspections.length > 0 ? (
```

Comment:
```suggestion
                            {allInspections.length > 0 ? (
```
allInspections cannot be undefined so this should be the same